packages/compass-welcome/src/components/welcome-tab.tsx

```diff
@@ -218,7 +218,7 @@ function WelcomeImage() {
 }
 
 export default function WelcomeTab() {
-  const { createNewConnection } = useConnections();
+  const { createNewConnection } = useConnectionActions();
```

Comment:
This one was a drive-by. I don't think there's any reason to use the deprecated useConnections() here.